readme.md

```diff
@@ -80,6 +80,15 @@ Default: `false`
 
 Do not bring the app to the foreground.
 
+##### newInstance <sup>(macOS only)</sup>
+
+Type: `boolean`\
+Default: `false`
+
+Open a new instance of the application(s) even if one is already running.
+
+On other platforms, this is always to open a new instance.
```

Comment:
This needs to be better written.

Reply:
Updated to `For other platforms, this value is always to be true.`
is this better?